### PR TITLE
Axis + error bars

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
This pull request should probably have an actual look-over before merging.

### CODE REVIEW

File: R/calc_views.R [hunk 1]
```diff
@@ -80,16 +80,47 @@ calc_view_usr <- function(views){
...
-      client_lims <- lims_from_client(side_components, var='ylim', side)
-      lims[!is.na(client_lims)] <- client_lims[!is.na(client_lims)]
-      usr <- usr_from_lim(lims, type=par()$yaxs)
+      usr <- usr_from_lim(lims, type=par()$yaxs) 
     } else {
       # is x
-      lims <- lims_from_list(lapply(side_components, var='x', function(list, var) strip_pts(list,var)))
-      client_lims <- lims_from_client(side_components, var='xlim', side)
-      lims[!is.na(client_lims)] <- client_lims[!is.na(client_lims)]
+      if(length(grep("^x$", unlist(lapply(side_components, names)))) > 0){
```
Comment (jordansread) [inline R/calc_views.R:105]: you fancy

Comment (ldecicco-USGS) [inline R/calc_views.R:105]: I think you mean hacky

File: R/overrideGraphics.R [hunk 2]
```diff
@@ -14,11 +14,16 @@ overrideGraphics <- function(name, object, ...) {
         params <- append(list(object), params)
     }
     
-    
-    defFun <- getFromNamespace(ifelse(existsFunction(paste0(name,".default")), paste0(name,".default"), name), 'graphics')
+    base.package <- ifelse(length(grep(pattern="error_bar", x=name)) > 0, "gsplot", "graphics")
+    defFun <- getFromNamespace(ifelse(existsFunction(paste0(name,".default")), paste0(name,".default"), name), base.package)
```
Comment (jordansread) [inline R/overrideGraphics.R:18]: we should make sure these are the same things that happen in config

File: R/overrideGraphics.R [hunk 3]
```diff
@@ -14,11 +14,16 @@ overrideGraphics <- function(name, object, ...) {
...
     
     names(params)[which(names(params) == "")] <- names(formals(defFun))[which(names(params) == "")]
     
-    do.call(getFromNamespace(name, "graphics"), params)
+    if(base.package == "gsplot"){
```
Comment (jordansread) [inline R/overrideGraphics.R:22]: why the change?

Comment (ldecicco-USGS) [inline R/overrideGraphics.R:22]: There was a recursive problem. It was pretty funky actually. So this is allowing us to make new functions (like error_bars), set a default and gsplot method.

File: R/overrideGraphics.R (hunk 2 above)
Comment (ldecicco-USGS) [inline R/overrideGraphics.R:18]: hmm...yeah, in my head I made this more generalized...apparently not in real life
